WebServices/TodoWCF/UWP/TodoWCF.UWP.csproj

```diff
@@ -88,10 +88,15 @@
     <UseDotNetNativeToolchain>true</UseDotNetNativeToolchain>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform" Version="6.0.12" />
-    <PackageReference Include="Xamarin.Forms" Version="3.6.0.220655" />
+    <PackageReference Include="Microsoft.NETCore.UniversalWindowsPlatform" Version="6.2.8" />
+    <PackageReference Include="Xamarin.Forms" Version="3.6.0.264807" />
```

Comment:
Can you use 3.6.0.220655 instead? All samples in the repo are on that version, which makes it easy to do a find/replace for future mass updates.

Reply:
I did downgrade this, however it resulted in errors (which is why I upgraded in the first place). After downgrade the UWP project threw this unhandled exception while trying to handle image rendering: System.TypeLoadException: Could not find Windows Runtime type 'Microsoft.Graphics.Canvas.UI.Xaml.CanvasImageSource'

This issue is discussed here: https://github.com/xamarin/Xamarin.Forms/issues/5188

The workaround required adding the Win2D.uwp package and upgrading the target framework version in the UWP project.